### PR TITLE
fix(EMS-1186): Your buyer -  Check your answers - Address multiple lines

### PR DESCRIPTION
**Pull request by Zainzzkk**
# Issue:
* Address was not showing on multiple lines on check your answers for your buyer

# Changes made: 
* Replaced `\r\n` with `<br/>` in ui to display address on multiple lines
* Enabled hot reloading for docker

### CODE REVIEW

File: src/ui/server/helpers/summary-lists/your-buyer/company-or-organisation/index.ts [hunk 1]
```diff
@@ -26,8 +26,11 @@ const {
  * @returns {Object}
  */
 export const generateAddressObject = (answers: ApplicationBuyer) => {
+  // replace new lines with line breaks to display in summary list
+  const address = answers[ADDRESS].replaceAll('\r\n', '<br />');
```
Comment (ttbarnes) [inline src/ui/server/helpers/summary-lists/your-buyer/company-or-organisation/index.ts:30]: Could this be a helper function so we can easily re-use in other places if required?

Also is there a possible scenario where there is only `\r` and no `\n`, and vice versa?

Comment (Zainzzkk) [reply src/ui/server/helpers/summary-lists/your-buyer/company-or-organisation/index.ts:30]: from the component, no.  All line breaks entered into the textarea seem to be` \r\n` if a new line is added

File: src/ui/server/test-mocks/mock-buyer.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ dotenv.config();
 
 const mockBuyer = {
   companyOrOrganisationName: 'Test name',
-  address: 'Test address',
+  address: 'Test \r\n address',
```
Comment (ttbarnes) [inline src/ui/server/test-mocks/mock-buyer.ts:7]: Not a blocker - a more complete/populated mock address e.g 3 lines country etc would be great

**Review by ttbarnes — APPROVED** [review]

File: src/ui/server/test-mocks/mock-buyer.ts (hunk 2 above)
Comment (Zainzzkk) [reply src/ui/server/test-mocks/mock-buyer.ts:7]: will do in next PR

**Pull request merged**